magda-web-client/src/Components/Dataset/DatasetDetails.js

```diff
@@ -63,21 +74,14 @@ class DatasetDetails extends Component {
                         </div>
                     </div>
 
-                    <div className="record-details__sidebar mui-col-sm-4">
-                        <div className="tags">
+                    <div className='record-details__sidebar mui-col-sm-4'>
```

Comment:
Can you run prettier over this file?

Reply:
@AlexGilleran it seems prettier won't pick up the inconsistent use of  " or ' used for className.
Has manually changed ' to "